nixos/tests/cagebreak.nix

```diff
@@ -0,0 +1,98 @@
+import ./make-test-python.nix ({ pkgs, lib, ...} :
+
+let
+  cagebreakConfigfile = pkgs.writeText "config" ''
+    workspaces 1
+    escape C-t
+    bind t exec env DISPLAY=:0 ${pkgs.xterm}/bin/xterm -cm -pc
```

Comment:
What annoys me about this is that the DISPLAY environment variable is `:1` inside the VM, but when I test cagebreak locally (started as a nested wayland session within Sway) the `DISPLAY` environment variable is properly set to `:0`.
This might be a bug with cagebreak being launched by an X11 session manager (lightdm).